src/elements/menu/menu/menu.visual.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { SbbBreakpointMediumMin } from '@sbb-esta/lyne-design-tokens';
+import { setViewport } from '@web/test-runner-commands';
+import { html } from 'lit';
+import { styleMap } from 'lit/directives/style-map.js';
+
+import type { SbbButtonElement } from '../../button/button.js';
+import { describeViewports, visualDiffDefault } from '../../core/testing/private.js';
+
+import '../../button/button.js';
+import '../../divider.js';
+import '../../link.js';
+import '../menu-link.js';
+import '../menu-button.js';
+import './menu.js';
+
+describe(`sbb-menu`, () => {
+  const userNameStyle = {
+    fontWeight: 'bold',
+    fontSize: 'var(--sbb-font-size-text-xs)',
+    marginTop: 'var(--sbb-spacing-fixed-1x)',
+  };
+
+  const userInfoStyle = {
+    color: 'var(--sbb-color-graphite)',
+    fontSize: 'var(--sbb-font-size-text-xxs)',
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'], viewportHeight: 400 }, () => {
+    it(
+      visualDiffDefault.name,
+      visualDiffDefault.with(async (setup) => {
+        await setup.withFixture(
+          html`
+            <sbb-button id="menu-trigger-1" size="m"> Menu trigger</sbb-button>
+            <sbb-menu trigger="menu-trigger-1">
+              <sbb-menu-link icon-name="link-small" href="https://www.sbb.ch/en">
+                View
+              </sbb-menu-link>
+              <sbb-menu-button icon-name="pen-small" amount="16" disabled> Edit </sbb-menu-button>
+              <sbb-menu-button icon-name="swisspass-small" amount="123"> Details</sbb-menu-button>
+              <sbb-divider></sbb-divider>
+              <sbb-menu-button icon-name="cross-small">Cancel</sbb-menu-button>
+            </sbb-menu>
+          `,
+          { minHeight: '400px' },
+        );
+        await setViewport({ width: SbbBreakpointMediumMin, height: 400 });
```

Comment:
```suggestion
        await setViewport(this.test?.ctx?.['requestViewport']);
```
Little suggestion 

 Ps.
From my understanding, you need to set the viewport before opening the menu to position it correctly which seems reasonable.
I think @jeripeierSBB mentioned a stability problem on webkit if we set the viewport before the fixture. Not sure if it applies here

Reply:
`this` is undefined, but i totally agree with you about moving the logic before the `withFixture`
the same issue is present in the notification and i possibly in every overlay component (hope not)